modules/bot_manager.py

```diff
@@ -18,7 +18,11 @@ def echo_all(message):
             parsed_text = message.text[1:]
             splitted_text = parsed_text.split(':')
             parsed_name, parsed_status = splitted_text[0][::-1].strip(), splitted_text[1][::-1].strip()
-            update_user_status(parsed_name, parsed_status, message.from_user.id, self.json_data)
+            try:
+                update_user_status(parsed_name, parsed_status, message.from_user.id, self.json_data)
+            except:
```

Comment:
If we catch here a specific exception, one for bad status, other for bad username, it will be easier to send an error message to the user (connected to the next comment)